src/common/components/paymentMethods/editPaymentMethod/editPaymentMethod.tpl.html

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
@OzzieOrca , @adammeyer , do you guys think it would be good to make bank account/credit card forms handle editing as well? In my example bellow I use the forms and virtually pass some payment method data that could be populated into the form.